packages/terra-dialog-modal/docs/README.md

```diff
@@ -0,0 +1,77 @@
+# Terra Dialog Modal
+
+The Dialog Modal would allow for dynamic height modals. Dialog modal would be a limited use case, as dynamic heights breaks with more complicated DOM structures. If content is too complicated, the terra-modal-manager should be used. The components should be placed above the 6000 z-index, but below the 8000 z-index. The dialog will need to support release and request focus props similar to terra-popup and terra-date-picker, so it can be presented from another modal with focus. The dialog modal will not support popups/date pickers, if those are needed use the terra-modal-manager.
```

Comment:
This component needs to support popups and date pickers.

Reply:
Updated the description.